### DISCUSSION TITLE
結婚後是否要生小孩?

### DISCUSSION THREAD

**Discussion by 111426027** [1101-GS4719-C/Python-Community-GS4719-C#1793] [General]
近年來台灣的生育率逐漸降低，少子化的情形愈來愈嚴重。想問大家未來結婚後，是否有意願想要生小孩?(個人是不想，覺得都快養不活自己了，還要花錢去照顧小孩，真的會吃不消。)

**Comment by mrweixd** [comment]
我覺得我個人不想生小孩，主要是基於對自身生活的期望和價值觀。我注重事業的發展、自我實現和追求個人興趣，而育兒的責任可能會影響到這些目標的實現。我認為不生小孩可以讓我更自由地追求自己的生活方式和目標。

**Comment by lyykey** [comment]
看到小孩一代不如一代，自己這代都要愁吃穿了，不敢想像未來有孩子的日子。可能是目前還在上學的年紀，對於未來仍然很迷茫，充滿著升學、以及之後賺錢的壓力，所以我認為或許之後可能會改觀，不過現在仍然沒有結婚生小孩的打算。

**Comment by SusuTima** [comment]
如果我有能力的話，會想要有小孩的陪伴，但如果我連自己都養不起，那就不會想要有小孩了。我希望我的小孩能在不愁吃穿的情況下快樂成長。

**Comment by benbebobo149** [comment]
我覺得如果我有能力的話，還是希望可以生小孩，我覺得養小孩算是一種對於童年缺陷的彌補，也可以重新當一次小孩。這個機會是蠻難得的，但是也要付出蠻大的代價，因此還是三思而後行會比較好。對我來說，擁有自己的孩子是一種美好的體驗，可以讓我們學會更多關於愛與責任的重要性，也能夠深入了解家庭的意義和價值。然而，我也意識到育兒不僅是一項責任，更是一項長期的承諾，需要我們有足夠的心理準備和經濟能力。因此，決定是否要生育子女是一個非常重要的決定，需要我們慎重考慮和衡量。最後，我要感謝同學的分享，讓我有機會思考到生育的意義和責任，這確實是一個值得深思的話題。

**Comment by LinChloe** [comment]
我覺得我個人想生小孩，主要是基於對自身生活的期望和價值觀。我注重事業的發展、自我實現和追求個人興趣，雖然育兒的責任可能會影響到這些目標的實現。但我認為在這之前需要具備能養好小孩的環境。

**Comment by thomasxshen** [comment]
這是一個非常個人的決定，並且可能會受到許多因素的影響，包括個人願望、伴侶意見、經濟狀況、生活條件、健康狀況等。有些夫妻選擇生育，認為孩子能為家庭帶來喜悅和滿足感；而有些夫妻則選擇不生育，可能是出於個人生活規劃或其他原因。重要的是與伴侶進行開放和誠實的溝通，共同做出最適合雙方的決定。如果您在這個問題上需要更多的指導或幫助，可能會考慮尋求專業人士，如婚姻顧問或心理學家的意見。

**Comment by yxin214** [comment]
我覺得生和不生各50 50。先說不想生的理由，純粹因為我覺得自己很難搞，真心覺得爸媽能把我養到現在很了不起，完全不知道我的小孩跟我一樣該怎麼養。但想生的理由，是我一直以為我爸很喜歡小孩子(他都會逗親戚的小小孩玩)，後來聽我媽說他是生了小孩之後才越來越喜歡小孩的，我想我自己生了小孩後心態也會改變？做個小總結，如果未來經濟能力允許，我應該會考慮生小孩。

**Comment by tzurou** [comment]
我自己是生和不生各占一半，如果生的話，就會是一場漫長的旅程，要養小孩、陪伴他們、甚至要花至少20年的時間和錢在小孩身上，但這會是人生的經驗，也會學習到很多不養小孩不會知道的事，年紀變大的時候和小孩處的好就可以當朋友也有個心靈的寄託，而不生小孩可以花很多時間在自己身上，做很多自己想做的事，但隨著年紀增長可能會覺得逐漸覺得空虛，畢竟現在平均年齡80歲，在這期間要不斷找自己想做的個人覺得蠻難的。

**Comment by YifenLin** [comment]
我並沒有太強烈一定要生或是不生小孩。目前的想法是比較傾向不生小孩，就算我以後有足夠的金錢養活自己，我會比較傾向於把金錢投資在自己身上，充實自己。像是運動，或培養一些其他興趣等等。抑或是提升自己的生活品質，安排旅遊或是住更好的房子。當然，不排除以後我的想法可能會改變，會想要和另一個人共組家庭之類的。

**Comment by wanka2030** [comment]
我認為小孩應該是能夠作為自身的依靠，結婚後是否要生小孩是一個非常個人化的決定，需要考慮到夫妻雙方的生活方式、價值觀、經濟能力等多個因素。有些夫妻可能希望生小孩來完整家庭，享受育兒的喜悅，而有些則可能選擇不生育，專注於事業或其他興趣。最重要的是夫妻雙方要充分溝通，共同確定對未來生育計劃的想法。但小孩不該成為婚後的附屬品，應該是婚姻的一員。

**Comment by hbh10601** [comment]
我個人是會偏向於生小孩，因為除了一些家庭的傳統期待之外，小孩對我來說有點像是和另一半的共同分身，如果想要讓彼此一直活著，除了保持身體健康以外，生小孩對我來說也是一個很有意義的事情。
另外，看到現在的社會現象，我很感謝爸媽有把我教好，這樣會讓我更想把我的下一代教好，回饋社會，為社會盡一份心力。

**Comment by Hahahahbla** [comment]
生育率下降的確是一個值得關注的議題。生小孩有其利有弊。生育帶來的喜悅和成就感是無可取代的，但也需考慮財務、時間和壓力等方面。對於個人而言，自主權至上，無論選擇生育與否，皆應受到尊重。

**Comment by Eileen050824** [comment]
我個人傾向生小孩，希望一個最多兩個(主要是現實考量)。我得承認養小孩很負擔也很牽絆，花錢的同時也失去一定的自由。但我認為雖然生活多了很多掛念，但換來的是珍貴的生活感受，還有同孩子成長的機會，最重要的是多一個讓自己去愛去守護的對象，這何嘗不是一種純粹的幸福？

**Comment by yyujimmy** [comment]
生育意願會因個人的價值觀、生活方式、經濟狀況等因素而有所不同，並不是每個人都有相同的想法。像你提到的，一些人可能會考慮到經濟壓力、生活壓力或者是對於個人生活的追求，而選擇不要生小孩。

另一方面，也有許多人希望組建家庭、體驗育兒的喜悅，並且相信小孩的出生能夠豐富他們的生活，帶來更多的幸福和意義。

在台灣以及全球範圍內，少子化的問題確實值得重視，因為這可能對社會和經濟產生各種影響。政府和社會應該提供更多支持和政策措施，以鼓勵有生育意願的夫妻，並提供更好的育兒環境和支持制度。

**Comment by willychiuchiu** [comment]
生育需要考慮到經濟、時間和生活壓力等因素，這是非常現實的考量。重要的是，無論選擇如何，都需要為自己的生活做出明智的決定，而社會也應提供支持和選擇的彈性，以滿足不同人群的需求。

**Comment by KH1126** [comment]
如果我的經濟能力能應付養育孩子的支出（包括娛樂費用），我會選擇擁有一個或最多兩個孩子。
我認為小孩子應該在一個穩定的環境中成長，而且他們的生活品質不應受到家長財務壓力的影響。
如果我真的有孩子，我會盡力讓他們體驗他們感興趣的活動，而不必擔心金錢問題。
我認為如果父母無法提供孩子所需的基本需求和機會，這將被視為失敗，因為生育孩子就意味著要為他們的生活負責。

**Comment by jim1743** [comment]
是否要生小孩，若是我的經濟能力能夠負擔養育小孩的費用，我會傾向擁有一個小孩，畢竟養一個孩子可慮的面向很多，並且十分消耗自己的精力，但最後的選擇還要和另一半溝通後，才能做出決定說到底要不要生。

**Comment by William08290829** [comment]
英國國神經科學家解釋說，從他們所獲得的數據顯示，還是有小孩能讓人更覺更加幸福：雖然有小孩也能讓人感到不快，或是讓人承受壓力和焦慮，但是總體來說，同無孩者相比，有孩子的人所經歷的情感體驗更強烈。

當然，科學家的結論歸結論，人們最終是否要小孩，還是要看個人的境遇與狀況以及眾多內部、外部、主管和客觀原因，絶非是否有幸福感就能決定的。

**Comment by YOGA012** [comment]
我覺得生不生小孩是個人的選擇，並非每個人都有同樣的願望或能力去承擔。重視自己的生活和未來是最重要的，而且不是每個人都希望或適合當父母。這樣的想法完全可以被尊重，而且越來越多的人都選擇不要生小孩，這也是一個社會上的趨勢。

**Comment by titan92601** [comment]
我認為生育意願因人而異，受經濟、生活壓力等多方面影響。個人選擇不生育或延後，反映現代價值觀與生活方式。然而，少子化對社會長遠發展構成挑戰，需要政府及社會共同因應。

**Comment by sunset513** [comment]
在我看來，生小孩幾乎只有壞處完全沒有好處。

小孩會增加生活的負擔，除了金錢，更要花費許多我精力在小孩上，而且小孩不一定能帶給我正向的回報。在這樣的狀況下，我寧願選擇養寵物。

當然，這只是我基於我的人生規劃而生的看法，也只適用於我。如果我的朋友很想要有小孩，我也會十分開心的祝福他。

**Comment by alexender0104** [comment]
對我來說是否要生小孩要看個人得經濟能力是否有辦法負擔，若無法負擔，我認為這樣只是在給孩子痛苦與困難，但這也只是我個人的想法，當夫妻二人有雙方愛的結晶時，也是一件不錯的事!

**Comment by 155Julia** [comment]
我覺得有能力的話一定要生，甚至如果經濟條件允許，我還想多生幾個，這樣家庭會更加熱鬧，很喜歡那種過年過節一大家人團聚圍爐的感覺，希望等哪天老了也會有很多子子孫孫到自己家來然後我發紅包給他們。而且養小孩也是一種投資，搞不好他們長大後會轉比我更多錢再回來孝盡我！

**Reply by benbebobo149** [reply]
是喔😯

**Comment by OG-Matcha** [comment]
以我目前的觀點來看，生小孩取決於一個人經濟是否能夠負擔、是否有長遠的規劃去安排這件事，如果連自己的生活都不理想的話，何苦生個小孩讓他和自己一起受難，也不是每個人都適合當父母、有正確的價值觀去教育小孩，最重要的還是夫妻要先有共識，確定對未來計劃的想法再做打算。

**Comment by sandyho0307** [comment]
我覺得是否生小孩受到多種因素的影響，如個人的經濟狀況、生活條件、伴侶意見等。其中，經濟狀況是一個重要因素。在現實經濟壓力下，教育、健康等方面的花費不斷增加，養育孩子的財務負擔可能會太重。

**Comment by cassy0201** [comment]
我覺得是否生孩子這個決定會隨著年齡、身分的轉變跟著變動，思考的現實層面也會受影響，是對生命負責的重要決定阿

**Comment by OHoln** [comment]
雖然不確定以後想法會不會改變，但我目前是認為我絕對不可能生小孩，一來是經濟的因素影響，二來是我不認為我有足夠的耐心來面對小孩，與其將他生下來讓他面對一段痛苦的家庭經歷，不如打一開始就不要生

**Comment by 710388** [comment]
生肯定生，小孩子非常可愛沒錯，帶小孩也很有成就感

**Comment by asd99263fg** [comment]
結婚的話我覺得要不要生小孩真的完全要看自己以及兩個人的意願，很多事情並不適自己想就可以，畢竟結婚是兩家子的人的事情，而且生小孩也是重大的一件跳戰，不要兩個人連薪水都沒有基礎就說要生小孩，只會讓孩子出生之後沒有好的童年造成社會更多問題，反倒是政府應該想想怎麼去解決這個少子化的問題鼓勵生育
